__tests__/app.test.js

```diff
@@ -263,6 +263,17 @@ describe("tests GET endpoint /api/articles/:article_id", () => {
         });
       });
   });
+  test("GET: 200 responds with the correct article object, with properties now including comment_count", () => {
+    return request(app)
+      .get("/api/articles/2")
+      .expect(200)
+      .then(({ body: { article } }) => {
+        expect(article.article_id).toBe(2);
+        expect(article).toMatchObject({
+          comment_count: expect.any(Number),
```

Comment:
This test tells me that you have a property called `comment_count`, but it doesn't tell me that the count is correct! 😄